src/js/jobs/components/List.jsx

```diff
@@ -28,7 +28,15 @@ const JobsListEmpty = styled(Box)`
 `;
 
 export const JobsList = ({ canArchive, canCancel, jobs, noJobs, onLoadNextPage, page, page_count, states }) => {
-    useEffect(() => onLoadNextPage(["preparing", "running"], 1), []);
+    const initialStates = ["preparing", "running"];
```

Comment:
Define about the component in module scope. This avoid the array being redefined every render.